getsolus/packages
# dosbox: Houskeeping

**Summary**

- Added appstream metadata (Part of getsolus/packages#1389)
- Added homepage key (Part of getsolus/packages#411)
- Added desktop file with StartupWMClass
- Install icon to hicolor instead of pixmaps

**Test Plan**

- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`
- Verify homepage loads relevant information
- Make sure app icon switch properly depending on icon theme

**Checklist**
